cassandra-3/src/main/java/io/debezium/connector/cassandra/Cassandra3SchemaChangeListener.java

```diff
@@ -44,6 +44,7 @@ public void onSessionReady(Session session) {
         List<TableMetadata> cdcEnabledTableMetadataList = getCdcEnabledTableMetadataList(session);
         for (com.datastax.oss.driver.api.core.metadata.schema.TableMetadata tm : cdcEnabledTableMetadataList) {
             schemaHolder.addOrUpdateTableSchema(new KeyspaceTable(tm), new KeyValueSchema(this.kafkaTopicPrefix, tm, this.sourceInfoStructMaker));
+            onKeyspaceCreated(session.getMetadata().getKeyspace(tm.getKeyspace().toString()).get());
```

Comment:
This seems to be yet another issue. Could you please create a Jira issue also for this bug and move it into separate commit. Btw. the same issue seems to be in the `Cassandra4SchemaChangeListener`.